extensions/wikia/CommunityPage/models/CommunityPageSpecialTopAdminsFormatter.class.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class CommunityPageSpecialTopAdminsFormatter {
+	const TOP_ADMINS_LIMIT = 10,
+		TOP_ADMINS_MODULE_LIMIT = 3;
+
+	/**
+	 * Returns array with fields to supply topAdmins.mustache template
+	 * If there are more than three admins provided returns first two and the count of remaining ones
+	 * @param array $topAdminsDetails
+	 * @return array
+	 */
+	public static function prepareData( array $topAdminsDetails ) {
+		$topAdminsCount = count( $topAdminsDetails );
+		$remainingAdminCount = self::prepareRemainingCount( $topAdminsCount );
+		return [
+			'admins' => self::prepareAdminsToShow( $topAdminsCount, $remainingAdminCount, $topAdminsDetails ),
+			'otherAdminCount' => $remainingAdminCount,
+			'haveOtherAdmins' => $remainingAdminCount > 0,
+			'adminCount' => $topAdminsCount,
```

Comment:
Shouldn't it be `topAdminsCount`?

Also, there's an inconsistency - some keys use the singular form `admin` and some the plural one `admins`. Can we change all of them to follow the same form?